.github/workflows/deploy-docs.yml

```diff
@@ -37,7 +38,12 @@ jobs:
             
             - name: Install graphviz
               run: sudo apt-get install graphviz -y
-    
+
+            - name: Clone Doxygen Awesome CSS Themes
+              working-directory: ./docs/doxygen
+              run: |
+                git clone --depth 1 -b v${{ env.DOXYGEN_AWESOME_VERSION }} https://github.com/jothepro/doxygen-awesome-css
```

Comment:
Nit: Simplify the name of the folder that git clones to to something like `awesome_css`?

Reply:
Done !!